src/ui/pages/BlogPage/DetailView.jsx

```diff
@@ -149,36 +159,42 @@ export class DetailView extends React.Component {
         </div>
 
         <div className={s.ContentDetailView}>
-          <h2 className={s.blogpostTitle}>{this.state.blogpost.title}</h2>
-          <div className={s.Autor}>{this.state.blogpost.autor}</div>
-          <h5 className={s.blogpostDate}>{datetime}</h5>
-          {reactElement}
-          <img
-            aria-label="blogpostImage"
-            alt=""
-            src={this.state.blogpost.image}
-          />
-        </div>
-
-        <div className={s.ButtonNavigation}>
-          {this.state.blogpost.previouspost && (
-              <a
-                  title="prevPost"
-                  href={`/blog/${this.state.slugPreviouspost}`}
-                  onClick={this.scrollToTop}
-              >
-                <button className={s.ButtonPrevious}>Previous</button>
-              </a>
-          )}
-          {this.state.blogpost.nextpost && (
-              <a
-                  title="nextPost"
-                  href={`/blog/${this.state.slugNextpost}`}
-                  onClick={this.scrollToTop}
-                  className={s.ButtonNext}
-              >
-                <button className={s.ButtonNextOnHover}>Next</button>
-              </a>
+          {!this.state.blogpost ? (
```

Comment:
This change should show the error page on every render (at least initially) we should adjust the way that we use to show the error page here